protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -1,15 +1,20 @@
 # Built-in [Product](./0051-PROD-product.md): Cash Settled Perpetual Futures (CSF)
 
-This built-in product provides perpetual futures that are cash-settled, i.e. they are margined and settled in a single asset.
+This built-in product provides perpetual futures contracts that are cash-settled, i.e. they are margined and settled in a single asset, and they never expire.
 
 [Background reading](https://www.paradigm.xyz/2021/05/everlasting-options/#Perpetual_Futures)
 
-Perpetual futures are a simple "delta one" product.
+Perpetual futures are a simple "delta one" product. Mark-to-market settlement occurs with a predefined frequency as per [0003-MTMK-mark_to_market_settlement](0003-MTMK-mark_to_market_settlement.md).  Additionally, a settlement using external data is carried out whenever `settlement_cue` is triggered AND the `settlement_data` is received within the specified `data_ingestion_period`. A number of protective measures can be specified for a market to deal with data scarcity in a predefined way.
 
 ## 1. Product parameters
 
-1. `settlement_data (Data Source: number)`: this data is used by the product to calculate periodic settlement cashflows. The receipt of this data triggers this calculation and the transfers between parties to "true up" to the external reference price.
 1. `settlement_asset (Settlement Asset)`: this is used to specify the single asset that an instrument using this product settles in.
+1. `settlement_cue (Data Source)`: this data is used to indicate that next periodic settlement should happen imminently.
+1. `settlement_data (Data Source: number)`: this data is used by the product to calculate periodic settlement cashflows. The receipt of this data triggers this calculation and the transfers between parties to "true up" to the external reference price.
```

Comment:
Specify how we want to filter the data so that we only consider a subset of it (based on filter).

Reply:
Added pseudocode: 
https://github.com/vegaprotocol/specs/pull/1504/files#diff-9c47979319e99ba93f2f121a920be94964b074266e01196f107414ce2121187dR25-R46